pkg/sql/plan/query_builder.go

```diff
@@ -125,6 +125,7 @@ func (builder *QueryBuilder) remapAllColRefs(nodeID int32, colRefCnt map[[2]int3
 		}
 
 		for i, col := range node.TableDef.Cols {
+			col.Typ.Table = node.TableDef.Name
```

Comment:
if Typ.autoIncr need to set in here?

Reply:
if you you set Table & AutorIncr in Resolve interface, that will not necessary to set value here.

Reply:
ok